.github/workflows/cmake.yml

```
@@ -21,11 +21,17 @@ jobs:
     - uses: actions/checkout@v4
       with:
         submodules: recursive
+    
+    - name: install_dependencies
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2006:style:1:66: Use $(...) notation instead of legacy backticks `...`

(shellcheck)